src/components/DropdownIcon/presets/0-default.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import DropdownIcon from "../DropdownIcon";
+
+export default (
+  <DropdownIcon
+    icon="more-vert"
```

Comment:
The preset is missing `uxpId`. It is very important for mapping the existing prototypes and components when deploying to production.